### PR TITLE
Allow docker compose to set port 

### PR DESCRIPTION
**Pull request by deepakmeena635**
Hi there, 

Currently, I'm deploying a moto container with other aws and localstack containers as others don't support athena.
It's being done with a docker-compose file which makes it easily manageable. But the problem is on some of my teams machines port 5000 is blocked by os and the process respawns when killed.

It'd be awesome if we can allow the port to be configured in the dockerfile. if there are other solutions that allow it to be done via docker compose I'll be more than happy to implement them.

### CODE REVIEW

**Comment by bblommers** [comment]
Hi @deepakmeena635, the server-script already has a `MOTO_PORT` environment variable that you can use:

https://github.com/getmoto/moto/blob/bbb07b4e7ede832d69b8fe0089e0707c915710fc/moto/server.py#L41

Does that fix your problem?

**Comment by deepakmeena635** [comment]
thanks @bblommers, it solves my problem is there any documentation that i can update for anyone else wondering about this

**Comment by mikegrima** [comment]
Is this ready to merge?

**Comment by deepakmeena635** [comment]
Thanks @bblommers and @mikegrima done from my side

File: Dockerfile
Comment (bblommers) [inline Dockerfile:1]: The changes in this file are not necessary, right?

**Review by bblommers — CHANGES_REQUESTED** [review]

File: Dockerfile
Comment (deepakmeena635) [reply Dockerfile:1]: yes one second

Comment (deepakmeena635) [reply Dockerfile:1]: done, thanks for pointing it out

**Comment by deepakmeena635** [comment]
wait

File: Dockerfile
Comment (deepakmeena635) [reply Dockerfile:1]: done now

**Review by bblommers — APPROVED** [review]
Awesome, thanks for adding this @deepakmeena635!

**Pull request merged**